shared/src/commonMain/kotlin/io/customer/shared/tracking/api/model/Device.kt

```diff
@@ -1,7 +1,10 @@
+@file:UseContextualSerialization(Any::class)
```

Comment:
Comment above this code explaining why this is here would be great.

Reply:
Sure, will add comments to it.